aten/src/ATen/native/native_functions.yaml

```diff
@@ -6389,19 +6389,38 @@
     CPU: sort_out_cpu
     CUDA: legacy::cuda::_th_sort_out
 
+- func: sort.values_stable(Tensor self, *, bool? stable, int dim=-1, bool descending=False, Tensor(a!) values, Tensor(b!) indices) -> (Tensor(a!) values, Tensor(b!) indices)
+  use_c10_dispatcher: hacky_wrapper_for_legacy_signatures
```

Comment:
Can this ditch the hacky wrapper?

Reply:
These new signatures look correct. In the future we should merge the sort signatures (more on that later).

Reply:
Shall I do it for all the `sort` methods, or just the newly introduced signatures?

Reply:
Tried, did not work because of the dispatch to the TH.